Doc/reference/executionmodel.rst

```diff
@@ -398,6 +398,82 @@ See also the description of the :keyword:`try` statement in section :ref:`try`
 and :keyword:`raise` statement in section :ref:`raise`.
 
 
+.. _execcomponents:
+
+Runtime Components
+==================
+
+Python's execution model does not operate in a vacuum.  It runs on a
+computer.  When a program runs, the conceptual layers of how it runs
+on the computer look something like this::
+
+   host computer (or VM or container)
+     process
+       OS thread (runs machine code)
+
+While a program always starts with exactly one of each of those, it may
+grow to include multiple of each.  Hosts and processes are isolated and
+independent from one another.  However, threads are not.
+
```

Comment:
This falls a little oddly: "multiples of each". Threads, sure. But to say that a (Python) program could grow to include multiple hosts or processes would take a more extended idea of "program" than I think the execution model seeks to address. Wouldn't it?

Reply:
Good point.  I'll narrow that.

Reply:
fixed